### ISSUE TITLE
Bug in seeder Command

### ISSUE THREAD

**Issue by oldpunk** [nWidart/laravel-modules#419]
`Nwidart\Modules\Commands\SeedCommand` in method 'handle'
```
try {
            if ($name = $this->argument('module')) {
                $name = Str::studly($name);
                $this->moduleSeed($this->getModuleByName($name));
            } else {
                $modules = $this->getModuleRepository()->getOrdered();
                array_walk($modules, [$this, 'moduleSeed']);
                $this->info('All modules seeded.');
            }
        } catch (\Exception $e) {
            $e = new FatalThrowableError($e);

            $this->reportException($e);

            $this->renderException($this->getOutput(), $e);

            return 1;
        }
```
`FatalThrowableError` accept `\Throwable` object, but if happens database error, it throw `Illuminate\Database\QueryException`

as result we have this error

```
[Symfony\Component\Debug\Exception\FatalThrowableError]
  Wrong parameters for Symfony\Component\Debug\Exception\FatalThrowableError([string $message [, long $code, [ long $severity, [ string $filename, [ long $
  lineno  [, Throwable $previous = NULL]]]]]])
```

**Comment by nWidart** [comment]
Seems to be an issue with the symfony version.

I still see the following signature on the symfony exception class `Symfony\Component\Debug\Exception\FatalThrowableError`:

```
public function __construct(\Throwable $e)
```

**Comment by oldpunk** [comment]
in which version it works correctly? 
I use laravel 5.5 
"nwidart/laravel-modules": "^2.5"
php7

**Comment by nWidart** [comment]
This was fixed in 2.7.

Sorry for the delay!
